PSReadLine/Completion.cs

```diff
@@ -209,6 +227,110 @@ private string GetUnambiguousPrefix(Collection<CompletionResult> matches, out bo
             return replacementText;
         }
 
+        private void DynamicHelpImpl(bool isFullHelp)
+        {
+            if (InViInsertMode())   // must close out the current edit group before engaging menu completion
+            {
+                ViCommandMode();
+                ViInsertWithAppend();
+            }
+
+            int cursor = _singleton._current;
+            string commandName = null;
+            string parameterName = null;
+
+            foreach(var token in _singleton._tokens)
+            {
+                if (token.TokenFlags == TokenFlags.CommandName)
+                {
+                    commandName = token.Extent.Text;
+                }
+
+                var extent = token.Extent;
+                if (extent.StartOffset <= cursor && extent.EndOffset >= cursor)
+                {
+                    if (token.Kind == TokenKind.Parameter)
+                    {
+                        parameterName = ((ParameterToken)token).ParameterName;
+
+                    }
+                    // Possible corner case here when cursor is at the end
+                }
+            }
+
+            if (!String.IsNullOrEmpty(commandName) && isFullHelp)
+            {
+                _ps ??= System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace);
+                _ps.Commands.Clear();
+
+                var fullHelp = _ps.AddScript($"(Microsoft.PowerShell.Core\\Get-Help -Name {commandName} -Full | Out-String)").Invoke<string>().FirstOrDefault();
```

Comment:
Please avoid using script if possible. It may not work in constrained language mode for VS Code.

Reply:
Fixed, used AddCommand instead.